engine/src/main/java/io/zeebe/engine/processing/job/JobThrowErrorProcessor.java

```diff
@@ -96,8 +95,7 @@ public void afterAccept(
     final var errorCode = job.getErrorCodeBuffer();
 
     final var foundCatchEvent = stateAnalyzer.findCatchEvent(errorCode, serviceTaskInstance);
-    // TODO (#6472) send TERMINATE_ELEMENT command instead
```

Comment:
Is #6472 resolved by this PR? If it's just related, please add it to the PR description to link the issue.